lib/decidim/friendly_signup/version.rb

```diff
@@ -3,7 +3,7 @@
 module Decidim
   # This holds the decidim-meetings version.
   module FriendlySignup
-    DECIDIM_VERSION = "0.27.0"
+    DECIDIM_VERSION = "0.27.4"
     COMPAT_DECIDIM_VERSION = "~> 0.27"
     VERSION = "0.4.4"
```

Comment:
We should bump the `VERSION` at least to 0.4.5 because we are changing dependencies

Reply:
Very true